invisible_cities/reco/corrections_new.py

```diff
@@ -200,10 +200,91 @@ def lt_xy_corrections(X : np.array, Y : np.array, Z : np.array, maps : ASectorMa
     LT  = get_maps_coefficient(X,Y)
     return correct_lifetime_(Z,LT)
 
+def get_df_to_z_converter(map_te: ASectorMap) -> Callable:
+    """
+    For given map, it returns a function that provides the conversion
+    from drift time to z position using the mean drift velocity.
+
+    Parameters
+    ----------
+    map_te : AsectorMap
+        Correction map with time evolution of some kdst parameters.
+
+    Returns
+    -------
+        A function that returns z converted array for a given drift time input
+        array.
+    """
+
+    dv_vs_time = map_te.t_evol.dv
+    dv         = np.mean(dv_vs_time)
+    def df_to_z_converter(dt):
+        return dt*dv
+
+    return df_to_z_converter
+
+
+class norm_strategy(AutoNameEnumBase):
+    mean   = auto()
+    max    = auto()
+    custom = auto()
+
+def get_pes_to_kev_factor(map_e0    : ASectorMap,
+                         norm_strat: norm_strategy   = norm_strategy.max,
+                         norm_value: Optional[float] = None              ) -> float:
+
+    if norm_strat is norm_strategy.max:
+        norm_value = amap_max(map_e0).e0
+    elif norm_strat is norm_strategy.mean:
+        norm_value = np.mean(np.mean(maps.e0))
+    elif norm_strat is norm_strategy.custom:
+        if norm_value is None:
+            raise MissingArgumentError
+    else:
+        raise ValueError
+    kr_energy  = 41.5575
```

Comment:
Add units here

Reply:
If units are used, this will be translating energies to the official unit in IC, that is MeV. Then, I guess that this function shouldn't be pes_to[whatever_unit] but it will always be refired to the official one, am I right?
What do the final users think? @paolafer @ausonandres

Reply:
If the function's name is `pes_to_keV`, it's correct to keep it as it is, as Brais says. If we multiply the number by the units, the function should be called `pes_to_MeV`.
Personally I'm a keV fan :-).

Reply:
I think it is a bit dangerous not to have units. This function will be used for example in esmeralda where a user has to define the threshold voxel energy, and if we dont have units here it means that the user can not specify units in the threshold energy parameter neither right?

Reply:
Yeah, you've got a point there. I'm afraid we need to be clear with the units.

Reply:
i think we should try not to have explicit unit in the function name, maybe something like to krypton_energy_scale or similar

Reply:
Fine with me.